subprojects/docs/src/docs/userguide/tutorial_using_tasks.adoc

```diff
@@ -15,6 +15,8 @@
 [[tutorial_using_tasks]]
 = Build Script Basics
 
+This chapter introduces you to the basics of writing Gradle build scripts. For a quick hands-on introduction, try the link:{guidesUrl}/creating-new-gradle-builds/[Creating New Gradle Builds] guide.
```

Comment:
Allows the removal of a direct link to the tutorial from the left-hand nav. It felt a little out of place and I want to minimise the number of navigation links to content outside of docs.gradle.org as it's quite disconcerting for the user.

Reply:
I don't disagree with the premise of removing this guide from the navigation, but I'm not sure that there's one natural place to add it and that this is that one place. 

Perhaps in the "Getting Started" page if it's not already there. Where else?

Reply:
It's already linked from the "Try Gradle" section of the Getting Started page. It is also the second getting started guide displayed when you follow the Tutorials link. I think that's sufficient.

Reply:
In that case perhaps we can just just remove this paragraph here, unless you think it fits well.